spec/spec_helper.rb

```diff
@@ -23,7 +23,7 @@ class Formatter
 require 'rubygems'
 require 'bundler/setup'
 
-Dir[File.join('./spec/support/*.rb')].each { |f| require f }
+Dir[File.join('./spec/support/*.rb')].sort.each { |f| require f }
```

Comment:
Any particular reason you are using sort here?
If not would you mind removing it?